django/db/migrations/state.py

```diff
@@ -255,6 +312,16 @@ def rename_field(self, app_label, model_name, old_name, new_name):
                         for to_field_name in to_fields
                     ])
         self.reload_model(app_label, model_name, delay=delay)
+        relations = copy.copy(self.relations)
+        model_key = (app_label, model_name)
+        for key, relation in relations.items():
+            if model_key in relation:
+                for field in self.relations[key][model_key]:
+                    if field[0] == old_name.lower():
+                        field[1].name = new_name.lower()
+                        self.relations[key][model_key].append((new_name.lower(), field[1]))
+                        self.relations[key][model_key].remove((field[0], field[1]))
+                        break
```

Comment:
Again there's a bit of unnecessary work here (alot of unnecessary `.lower()` calls that can be cached in local variables)